app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt

```diff
@@ -17,9 +17,9 @@ class AppVersionViewModel @Inject constructor(
   private val oppiaDateTimeFormatter: OppiaDateTimeFormatter
 ) : ObservableViewModel() {
 
-  private lateinit var context: Context
+  private val context: Context = fragment.activity!!.applicationContext
```

Comment:
Prefer injecting the application context if you can (which is possible in this case since the view model constructor is an injection constructor).

Also note that this could fail--fragments are not guaranteed to have an activity (hence the need for the ``!!``). We can, however, always inject an application context & it will be present.

Reply:
Okay! Makes sense.